pkgs/tools/text/align/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildGoPackage, fetchFromGitHub }:
+
+let
+  pname = "align";
+  version = "1.1.3";
+  rev = "v${version}";
+  owner = "Guitarbum722";
+in buildGoPackage rec {
+  inherit pname version rev;
+
+  goPackagePath = "github.com/${owner}/${pname}";
+
+  src = fetchFromGitHub {
+    inherit owner rev;
+    repo = pname;
+    sha256 = "17gs3417633z71kc6l5zqg4b3rjhpn2v8qs8rnfrk4nbwzz4nrq3";
+  };
+
+  doCheck = true;
+
```

Comment:
```suggestion
```

Reply:
why?

Reply:
`buildGoModule` defaults to `doCheck = true`
https://github.com/NixOS/nixpkgs/blob/3360a376cb905ac40e77d5424089d4893e9cd157/pkgs/development/go-modules/generic/default.nix#L236

Reply:
ah! good to know... I know it's disabled on most builds. Yay go.